lib/experimental/interactivity-api/directive-processing.php

```diff
@@ -329,6 +358,8 @@ function gutenberg_interactivity_evaluate_reference( $path, array $context = arr
 	 * E.g., "file" is an string and a "callable" (the "file" function exists).
 	 */
 	if ( $current instanceof Closure ) {
+		// TODO: Do not pass the store as argument. Implement something similar
+		// to what we have in the JS runtime (`getContext()`, etc.).
```

Comment:
@luisherranz, do you remember the idea we had for computing the derived state in the server? I added this comment just in case we want to change the current implementation (I'm pretty sure we want). 🤔

Reply:
Let's remove support for closures for now.